zello-sdk-sample-signin/build.gradle

```diff
@@ -5,34 +5,35 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.0'
+        classpath "com.android.tools.build:gradle:3.6.3"
     }
 }
 
-apply plugin: 'com.android.application'
+apply plugin: "com.android.application"
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
 
     defaultConfig {
-        applicationId "com.zello.sdk.sample.signin"
+        applicationId "com.zello.sdk.sample"
         minSdkVersion 15
-        targetSdkVersion 28
+        targetSdkVersion 29
         versionCode 1
         versionName "1.0"
     }
 
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
     }
 }
 
 dependencies {
-	implementation fileTree(include: '*.jar', dir: 'libs')
-	implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation fileTree(include: "*.jar", dir: "libs")
+    implementation "androidx.appcompat:appcompat:1.1.0"
+    implementation "com.android.support:support-annotations:28.0.0"
```

Comment:
You appear to be migrating the project to androidx, but you're still referencing some legacy support dependencies. This line, for example, could be replaced with `implementation 'androidx.annotation:annotation:1.1.0'`